paradise.dme

```diff
@@ -1822,6 +1822,7 @@
 #include "code\modules\shuttles\shuttle_supply.dm"
 #include "code\modules\shuttles\shuttles_multi.dm"
 #include "code\modules\shuttles\whiteship.dm"
+#include "code\modules\space transition\space_transition.dm"
```

Comment:
I don't see why it'd cause any issue. We already have several folders with spaces in them, and /tg/ uses the same.